src/Congo.WebApi/Startup.cs

```diff
@@ -24,8 +28,27 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddAppServices(Configuration);
-
             services.AddControllers();
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy(_CORS_DEV, policy =>
+                            policy.AllowAnyOrigin().AllowAnyMethod().AllowAnyHeader());
+
+                options.AddPolicy(_CORS_PROD, policy =>
+                {
+                    //TODO: Multiple Origins from env vars
+                    string ALLOWED_ORIGIN = Environment.GetEnvironmentVariable("ALLOWED_ORIGIN");
+                    if (ALLOWED_ORIGIN != null)
+                    {
+                        //Debug.WriteLine("ALLOWED_ORIGIN = " + ALLOWED_ORIGIN);
+                        policy.WithOrigins(ALLOWED_ORIGIN).AllowAnyMethod().AllowAnyHeader();
+                    }
+                    else if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Development")
+                        // to prevent this exception in dev env
+                        throw new Exception("No value set for env ALLOWED_ORIGIN.");
```

Comment:
I'd like to see a slightly more verbose error message and comment, and don't forget the braces :)

```suggestion
                    else if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Development")
                    {
                        // Don't throw any exceptions in development, as this policy is never used there
                        throw new Exception("No value set for mandatory environment variable ALLOWED_ORIGIN.");
                    }
```